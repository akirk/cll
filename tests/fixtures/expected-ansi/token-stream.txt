[1mTitle: The Last Whisper[m

In a [1mforgotten village[m, a [1myoung girl[m discovers an [1mancient book[m that holds the [1msecrets of lost voices[m. Each page unlocks a [1mwhisper from the past[m. As she speaks the words, long-buried [1mmemories[m awaken, revealing an [1mepic tale of love[m, [1mbetrayal[m, and the [1mpower of memories[m to change the future.