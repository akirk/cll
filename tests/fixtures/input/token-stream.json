["**","Title",":"," The"," Last"," Whisper","**\n\n","In"," a"," **","forgot","ten"," village","**,"," a"," **","young"," girl","**"," discovers"," an"," **","anc","ient"," book","**"," that"," holds"," the"," **","se","crets"," of"," lost"," voices","**","."," Each"," page"," unlock","s"," a"," **","wh","isper"," from"," the"," past","**","."," As"," she"," speaks"," the"," words",","," long","-b","ur","ied"," **","mem","ories","**"," awaken",","," revealing"," an"," **","ep","ic"," tale"," of"," love","**,"," **","betr","ay","al","**,"," and"," the"," **","power"," of"," memories","**"," to"," change"," the"," future","."]