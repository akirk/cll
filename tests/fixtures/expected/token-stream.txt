Title: The Last Whisper

In a forgotten village, a young girl discovers an ancient book that holds the secrets of lost voices. Each page unlocks a whisper from the past. As she speaks the words, long-buried memories awaken, revealing an epic tale of love, betrayal, and the power of memories to change the future.